simple/sample/output/triples.csv

```diff
@@ -15,15 +29,23 @@ var1,typeOf,StatisticalVariable,
 var1,name,,Good var1 name
 var1,description,,Good var1 description
 var1,memberOf,custom/g/group_2,
+var1,includedIn,custom/p/Prov_1,
```

Comment:
should there be a "includedIn" triple between a stat var group and source/prov?

Reply:
SV groups may have SVs in multiple sources / provenances. Is multiple `includedIn` triples the way to support this?

e.g.

```
svg1,includedIn,source1,
svg1,includedIn,provenance1,
svg1,includedIn,source2,
svg1,includedIn,provenance2,
```

Reply:
Yes!

Reply:
SG.

Should we support multiple sources for SVs as well?

Reply:
Yes, should support that

Reply:
Done. Added support for multiple sources and provenances in SVs and SVGs.